packages/fxa-auth-db-mysql/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.148.2
+
+### Other changes
+
+* release: Merge branch 'train-147' into train-148-merge-147 (66e170d45)
```

Comment:
These merge branches shouldn't really be added to the changelog. We can resolve that in another PR.